hack/upgrade-test.sh

```diff
@@ -42,14 +42,15 @@
 # The hyperconverged-cluster deployment's image is also checked
 # to verify that it is updated to the new operator image from 
 # the local registry.
-source hack/common.sh
 
-if [ -n "${KUBEVIRT_PROVIDER}" ]; then
+echo "KUBEVIRT_PROVIDER: $KUBEVIRT_PROVIDER"
+
+if [ -n "$KUBEVIRT_PROVIDER" ]; then
   # okd-* provider, STDCI
   REGISTRY_IMAGE="registry:5000/kubevirt/hco-registry"
   REGISTRY_IMAGE_UPGRADE="registry:5000/kubevirt/hco-registry-upgrade"
   REGISTRY_IMAGE_URL_PREFIX="registry:5000/kubevirt"
-  CMD="./cluster-up/kubectl.sh"
+  export CMD="./cluster-up/kubectl.sh"
```

Comment:
What do you think about putting the if else in a config file somewhere that is `source`'d by files that need it?

Reply:
Sure, let me revise this PR.